.buildkite/pipeline.yml

```diff
@@ -11,3 +11,8 @@ steps:
 
   - label: Build the pex file
     command: mkdir -p dist && .buildkite/build_pex.sh
+
+  - wait
```

Comment:
I already removed it @aron. And its build fine at [buildkite](https://buildkite.com/learningequality/kolibri/builds/430#f6d0d76c-db42-42f2-bab6-e7e2eeb3773b)